ConeSearch.tex

```diff
@@ -385,97 +522,69 @@ \subsubsection{Interface}
-to describe a service's support for the Simple Cone Search protocol;
-it is defined as follows:
+\subsubsection{\textit{cs:}ConeSearch} \label{subsec:cstype} The
+\xmlel{cs:ConeSearch} type is a \xmlel{vr:Capability} sub-type that
+should be used to describe a service's support for the Simple Cone
+Search protocol; it is defined as follows:
 
 % GENERATED: !schemadoc ConeSearch-v1.1.xsd ConeSearch
-\begin{generated}
-\begingroup
-      	\renewcommand*\descriptionlabel[1]{%
-      	\hbox to 5.5em{\emph{#1}\hfil}}\vspace{2ex}\noindent\textbf{\xmlel{cs:ConeSearch} Type Schema Documentation}
+\begin{generated} \begingroup \renewcommand*\descriptionlabel[1]{% \hbox
+to
+5.5em{\emph{#1}\hfil}}\vspace{2ex}\noindent\textbf{\xmlel{cs:ConeSearch}
+Type Schema Documentation}
 
-\noindent{\small
-            The capabilities of a Cone Search implementation.  
-         \par}
+\noindent{\small The capabilities of a Cone Search implementation.
+\par}
 
-\vspace{1ex}\noindent\textbf{\xmlel{cs:ConeSearch} Type Schema Definition}
+\vspace{1ex}\noindent\textbf{\xmlel{cs:ConeSearch} Type Schema
+Definition}
 
 \begin{lstlisting}[language=XML,basicstyle=\footnotesize]
-<xs:complexType name="ConeSearch" >
-  <xs:complexContent >
-    <xs:extension base="vr:Capability" >
-      <xs:sequence >
-        <xs:element name="maxSR" type="xs:float" minOccurs="0" 
-          maxOccurs="1" />
-        <xs:element name="maxRecords" type="xs:positiveInteger" 
-          minOccurs="0" maxOccurs="1" />
-        <xs:element name="verbosity" type="xs:boolean" />
-        <xs:element name="testQuery" type="cs:Query" minOccurs="0"
-          maxOccurs="1" />
-      </xs:sequence>
-    </xs:extension>
-  </xs:complexContent>
-</xs:complexType>
+<xs:complexType name="ConeSearch" > <xs:complexContent > <xs:extension
+base="vr:Capability" > <xs:sequence > <xs:element name="maxSR"
+type="xs:float" minOccurs="0" maxOccurs="1" /> <xs:element
+name="maxRecords" type="xs:positiveInteger" minOccurs="0" maxOccurs="1"
+/> <xs:element name="verbosity" type="xs:boolean" /> <xs:element
+name="testQuery" type="cs:Query" minOccurs="0" maxOccurs="1" />
+</xs:sequence> </xs:extension> </xs:complexContent> </xs:complexType>
 \end{lstlisting}
 
-\vspace{0.5ex}\noindent\textbf{\xmlel{cs:ConeSearch} Extension Metadata Elements}
+\vspace{0.5ex}\noindent\textbf{\xmlel{cs:ConeSearch} Extension Metadata
+Elements}
 
 \begingroup\small\begin{bigdescription}\item[Element \xmlel{maxSR}]
-\begin{description}
-\item[Type] floating-point number: \xmlel{xs:float}
-\item[Meaning] 
-                        The largest search radius, in degrees, that will be
-                        accepted by the service without returning an error 
-                        condition.  Not providing this element or 
-                        specifying a value of 180 indicates that there
-                        is no restriction.
+\begin{description} \item[Type] floating-point number: \xmlel{xs:float}
```

Comment:
The layout in the preview shows spurious `to 5.5em` at the start of each \item entry.